book.toml

```diff
@@ -31,7 +31,6 @@ verbose = false # Report timing information.
 
 [output.pandoc]
 optional = true
-hosted-html = "https://google.github.io/comprehensive-rust/"
```

Comment:
Was this removed on purpose? If not, please restore :)

Reply:
Done. I don't know how it got removed.